index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@200&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css" />
+    <link rel="stylesheet" href="./styles/utility.css">
+    <link rel="stylesheet" href="./styles/style.css">
+    <title>My portfolio</title>
+</head>
+<body>
+    
+    <header>
+        <div class="wrapper">
+            <div class="main-menu">
+                <div class="logo">
+                    <a href="#home">
+                        <h2>Welcome</h2>
+                    </a>
+                </div>
+                <nav>
+                    <!-- links will come here -->
+                    <div class="hamburger">
+                        <span class="bar"></span><span class="bar"></span><span class="bar"></span>
+                    </div>
+                </nav>
+            </div>
+        </div>
+    </header>
+    <section class='hero'>
+        <div class="wrapper">
+            <div class="headline-desc">
+                <h1>Hey there. I’m Mariane<br />
```

Comment:
[OPTIONAL]

- [ ] Please change the name `Mariane `to your name.

Reply:
Thanks for approving my PR, I appreciate 😄